Source/Main.cpp

```diff
@@ -0,0 +1,14 @@
+#include <bits/stdc++.h>
+#include "header.h"
+using namespace std;
+
+
+int main()
+{
+    cout << "Enter any four digit number : ";
+    long long int fourDigitNumber;
```

Comment:
Why are you taking input in int what if the user entered any alphabet or special character.